records_mover/cli/job_config_schema_as_args_parser.py

```diff
@@ -2,7 +2,7 @@
 from records_mover.utils.structures import nest_dict
 import os
 import json
-from ..types import JsonSchema, JobConfig
+from ..mover_types import JsonSchema, JobConfig
```

Comment:
I hit a situation where something got jostled and the local module named 'types' started conflicting with the global module named 'types'--I finessed the situation by renaming it: https://circleci.com/gh/bluelabsio/records-mover/6860